### DISCUSSION TITLE
Remove connected phone number from the employer's vercel account that is no longer active

### DISCUSSION THREAD

**Discussion by vignesh-web-dev** [vercel/community#3476] [Help] [answered]
### Summary

I quit my job at the previous company and my phone number is connected to the company's vercel account. Now I'm not able to log in with the company's id and also not able to log in with my personal account without that phone number. is there any way to resolve this? FYI, the company's email id is already deleted.

### Steps to Reproduce

I quit my job at the previous company and my phone number is connected to the company's vercel account. Now I'm not able to log in with the company's id and also not able to log in with my personal account without that phone number. is there any way to resolve this? FYI, the company's email id is already deleted.

**Comment by amyegan** [comment, marked as answer]
Hey @vignesh-web-dev. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email, username, and phone number for the accounts. Providing additional information about your account and the project you are deploying can help accelerate the verification process.
